src/Integration/Laravel/Mail/composer.json

```diff
@@ -13,8 +13,8 @@
     "require": {
         "php": "^7.2.5 || ^8.0",
         "async-aws/ses": "^1.0",
-        "illuminate/mail": "^6.18.13 || ^7.10 || ^8.0",
-        "illuminate/support": "^6.18.13 || ^7.10 || ^8.0"
+        "illuminate/mail": "^6.18.13 || ^7.10 || ^8.0 || ^9.0 || ^10.0",
```

Comment:
this doesn't work. the laravel 9 and 10 mailer are massively different.

Reply:
then the CI should be changed because right now, the root-dev job doesn't test with recent versions of Symfony because of those.